pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/api/Utils.kt

```diff
@@ -1,14 +1,17 @@
 package no.nav.pensjon.brev.api
 
-import no.nav.pensjon.brev.template.*
-import no.nav.pensjon.brevbaker.api.model.*
+import no.nav.pensjon.brev.api.model.TemplateDescription
+import no.nav.pensjon.brev.template.Language
+import no.nav.pensjon.brev.template.LetterTemplate
+import no.nav.pensjon.brevbaker.api.model.LanguageCode
 
 fun LetterTemplate<*, *>.description() =
     TemplateDescription(
         name = name,
         letterDataClass = letterDataType.java.name,
         languages = language.all().map { it.toCode() },
         metadata = letterMetadata,
+        kategori = null,
```

Comment:
Skal kategori være null for redigerbare brev også?

Reply:
Vil pesys tåle at vi returnerer ett ekstra felt for kategori som er null?

Reply:
Den er ikke null for redigerbarebrev, denne funksjonen brukes bare i v1 grensesnittet. 
Brevbaker-resttemplate i pesys har `disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)`.